code/frameworks/nextjs/src/config/webpack.ts

```diff
@@ -36,16 +36,16 @@ export const configureConfig = async ({
     addScopedAlias(baseConfig, 'react', 'next/dist/compiled/react');
   }
   if (tryResolve('next/dist/compiled/react-dom/cjs/react-dom-test-utils.production.js')) {
-    setAlias(
+    addScopedAlias(
```

Comment:
`addScopedAlias` uses `scopedResolve` internally, which effectively strips out the last bit of the file and `next/dist/compiled/react-dom/cjs/react-dom-test-utils.production.js` would resolve to `<absolute-path>/next/dist/compiled/react-dom/cjs`, which is not what we want.

So `addScopedAlias` and `scopedResolve` have to be rewritten to respect `id`'s which point to an exact file, instead of just a folder or module.

Reply:
Thanks! On it.

Reply:
@valentinpalkovic I think I've got this implemented correctly now, but I don't have a dev setup to test it.

If you've got any tips on how to run a `storybook --debug-webpack` against the @storybook/next framework in local dev, I'd gratefully accept them and do some testing myself. If not, happy for you to test this for me. 🙇🏻